cfme/exceptions.py

```diff
@@ -13,6 +13,26 @@ class CFMEException(Exception):
     pass
 
 
+class ConsoleNotSupported(CFMEException):
+    """Raised by functions in :py:mod:`cfme.configure.configuration` when an invalid
+    console type is given"""
+    def __init__(self, version):
+        self.version = version
+
+    def __str__(self):
+        return "Console not supported on less than Miq 5.8: current version {}".format(self.version)
```

Comment:
**Required**
Miq 5.8 doesn't exist, those are CFME version numbers.

Either pass version and product_name from appliance, or just state its not supported on current version.